modules/alloy/src/main/ts/ephox/alloy/ui/slider/SliderParts.ts

```diff
@@ -90,6 +91,8 @@ const thumbPart = PartType.required<SliderDetail, { dom: OptionalDomSchema; even
   }
 });
 
+const isShift = (event: any) => event.raw ? event.raw.shiftKey : false;
```

Comment:
Never use `any` in production code if there is a proper type. In this case you can do the following:

1. Use the `KeyMatch.isShift` function from the KeyMatch module import it as a named import
2. Change this to `const isShift = (evt: NativeSimulatedEvent<KeyboardEvent>) => evt.event.raw.shiftKey;` and remove the multiple calls to `event.event` (DRY)
3. Or do both make a isShift function that just calls `KeyMatch.isShift` but unpacks the event.event inside it.

- [ ] Remove the use of the `any` type or remove the predicate